static_src/util/cf_api.js

```diff
@@ -262,7 +262,7 @@ export default {
   postAppRestart(appGuid) {
     return http.post(`${APIV}/apps/${appGuid}/restage`).then(() => {
       appActions.restarted(appGuid);
-    });
+    }).catch((err) => handleError(err, appActions.error.bind(this, appGuid)));
```

Comment:
why the `handleError`? if we're handling the error via an action, doesn't it mean we're handling the error twice?